src/common/MetaPreview/MetaPreview.js

```diff
@@ -179,36 +181,63 @@ const MetaPreview = ({ className, compact, name, logo, background, runtime, rele
                             />
                         ))
                 }
-            </div>
-            <div className={styles['action-buttons-container']}>
-                {
-                    typeof toggleInLibrary === 'function' ?
-                        <ActionButton
-                            className={styles['action-button']}
-                            icon={inLibrary ? 'remove-from-library' : 'add-to-library'}
-                            label={compact ? null : inLibrary ? t('REMOVE_FROM_LIB') : t('ADD_TO_LIB')}
-                            tabIndex={compact ? -1 : 0}
-                            onClick={toggleInLibrary}
-                        />
-                        :
-                        null
-                }
                 {
-                    typeof trailerHref === 'string' ?
-                        <ActionButton
-                            className={styles['action-button']}
-                            icon={'trailer'}
-                            label={compact ? null : t('TRAILER')}
-                            tabIndex={compact ? -1 : 0}
-                            href={trailerHref}
-                        />
+                    !compact && typeof description === 'string' && description.length > 0 ?
+                        <div className={styles['description-container']}>
+                            <div className={styles['label-container']}>
+                                {t('SUMMARY')}
+                            </div>
+                            {description}
+                        </div>
                         :
                         null
                 }
+            </div>
+            <div className={styles['action-buttons-container']}>
+                <div className={styles['action-button-wrapper']}>
+                    {
+                        typeof toggleInLibrary === 'function' ?
+                            <ActionButton
+                                className={styles['action-button']}
+                                icon={inLibrary ? 'remove-from-library' : 'add-to-library'}
+                                label={compact ? null : inLibrary ? t('REMOVE_FROM_LIB') : t('ADD_TO_LIB')}
+                                tabIndex={compact ? -1 : 0}
+                                onClick={toggleInLibrary}
+                            />
+                            :
+                            null
+                    }
+                    {
+                        typeof toggleInLibrary === 'function' && compact ?
+                            <div className={styles['label']}>{inLibrary ? t('REMOVE_FROM_LIB') : t('ADD_TO_LIB')}</div>
```

Comment:
Those labels create a big gap under the buttons on discover
Either we create a tooltip system or maybe they should be placed on top positioned with absolute